src/Fixer/Phpdoc/PhpdocArrayStyleFixer.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\DocBlock\TypeExpression;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class PhpdocArrayStyleFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    private const STRATEGY_FROM_ARRAY_TO_LIST = 'array_to_list';
+    private const STRATEGY_FROM_BRACKETS_TO_ARRAY = 'brackets_to_array';
+    private const STRATEGY_FROM_BRACKETS_TO_LIST = 'brackets_to_array_to_list';
```

Comment:
I am wondering why it's not like this:

```suggestion
    private const STRATEGY_FROM_BRACKETS_TO_LIST = 'brackets_to_list';
```

I am not sure we should include intermediate state in the strategy's name, I would rather use source-target convention.